en/Developer policies.md

```diff
@@ -10,46 +10,46 @@ Plugins and themes must not:
+- Insert static ads outside a plugin’s interface.
 - Include client-side telemetry.
 - Themes may not load assets from the network. To bundle an asset, see [[Embed fonts and images in your theme|this guide]].
 
 ### Disclosures
 
-The following are only allowed if clearly indicated in your README:
+The following are only allowed if indicated in your README:
 
 - Payment is required for full access.
 - An account is required for full access.
 - Network use. Clearly explain which remote services are used and why they're needed.
 - Accessing files outside of Obsidian vaults. Clearly explain why this is needed.
-- Static ads such as banners and pop-up messages within the plugin's own interface.
-- Server-side telemetry. Link to a privacy policy that explains how the data is handled must be included.
-- Close sourced code. This will be handled on a case by case basis.
+- Static ads, including banners and pop-up messages within the plugin's interface.
+- Server-side telemetry. A link to a privacy policy that explains how the data is handled must be included.
+- Close-sourced code. This will be handled on a case-by-case basis.
 
 ### Copyright and licensing
 
 All community plugins and themes must follow these requirements:
 
-- Include a [LICENSE file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) and clearly indicate the license of your plugin or theme.
-- Comply with the original licenses of any code your plugin or theme makes use of, including attribution in the README if required.
+- Include a [LICENSE file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) and clearly indicate your plugin's license or theme.
+- Comply with the original licenses of any code your plugin or theme uses, including attribution in the README if required.
 - Respect Obsidian's trademark policy. Don't use the "Obsidian" trademark in a way that could confuse users into thinking your plugin or theme is a first-party creation.
 
 ## Reporting violations
 
-If you encounter a plugin or theme that violates the policies above, please let the developer know by opening a GitHub issue in their repository. Kindly check existing issues to see if it’s already reported.
+If you encounter a plugin or theme that violates the policies above, please inform the developer by opening a GitHub issue in their repository. Kindly check existing issues to see if it’s already been reported.
 
-If the developer doesn’t response after 7 days, [contact the Obsidian team](https://help.obsidian.md/Help+and+support#Report+a+security+issue). For serious violations, you can contact our team immediately.
+[Contact the Obsidian team](https://help.obsidian.md/Help+and+support#Report+a+security+issue) if the developer doesn’t respond after seven days. For serious violations, you can contact our team immediately.
```

Comment:
This reads as if there are two options, contacting the developer, or contacting us.

We would prefer it if users try to contact the developer first, and only then contact us.